dm/worker/status.go

```diff
@@ -203,7 +203,9 @@ func statusProcessResult(pr *pb.ProcessResult) *pb.ProcessResult {
 	result := proto.Clone(pr).(*pb.ProcessResult)
 	if result != nil {
 		for i := range result.Errors {
-			result.Errors[i].Error = nil
+			if result.Errors[i].Error != nil {
+				result.Errors[i].Msg = ""
```

Comment:
Can we use `TError` to replace `ProcessError` later?

then we can make 

```
                     "errors": [
                            {
                                "msg": "",
                                "error": {
                                    "ErrCode": 10006,
                                    "ErrClass": 1,
                                    "ErrScope": 2,
                                    "ErrLevel": 3,
                                    "Message": "this is the error message",
                                    "RawCause": "this is the raw cause",
                                    "Workaround": "this is the workaround"
                                }
                            }
                        ],
```

become

```
                     "errors": [
                            {
                                "ErrCode": 10006,
                                "ErrClass": 1,
                                "ErrScope": 2,
                                "ErrLevel": 3,
                                "Message": "this is the error message",
                                "RawCause": "this is the raw cause",
                                "Workaround": "this is the workaround"
                            }
                        ],
```

Reply:
BTW, if we can make `ErrClass`, `ErrScope` and `ErrLevel` to be shown as `string` rather than `int`, it must be better.